### ISSUE TITLE
Video editable: If video is deleted, a value is still stored

### ISSUE THREAD

**Issue by ckemptner** [pimcore/pimcore#2884]
Latest build (243)

If a video is added to a video editable in an object, and then deleted, the value "/" is still stored. In the frontend, this leads to problems because it makes workarounds necessary to prevent the container of the video from being loaded. Please fix this.

**Comment by Cruiser13** [comment]
I can confirm that this issue does still exist. If a user does delete the video from an object, it'll automatically set type asset and reference the / asset folder. Expecting NULL instead in this case.

**Comment by brusch** [comment]
Seems to be fixed already, wasn't able to reproduce this anymore. 

What I did: 

1. Created a class with a video field
![image](https://user-images.githubusercontent.com/142037/68400126-cd943680-0177-11ea-8a04-6e1cc3d8adcd.png)

2. Created a new object
3. Assigned an asset video to the video field
4. Saved the object
5. Removed the video from the field
![image](https://user-images.githubusercontent.com/142037/68400241-fae0e480-0177-11ea-81c2-41b442250039.png)

6. Run the following code
```php
$obj = DataObject::getById(1128);
$video = $obj->getVideo();
```

Output was: `null`
